plainly-plugin/src/ui/components/form/UploadForm.tsx

```diff
@@ -206,6 +206,13 @@ export function UploadForm() {
                 </div>
               ))}
             </div>
+            {!remoteProjectExists && (
+              <Alert
+                title="Local project is not linked to any project on the platform. If you have one, you can link it on the Projects page, and re-upload existing, or new one will be uploaded."
```

Comment:
We already have this note here:

<img width="392" alt="image" src="https://github.com/user-attachments/assets/fa2a4bad-2d07-49f1-87e7-79ac6bebdbdc" />

Maybe we should use the same or similar copy?

```Working project is not linked to any project on the Plainly platform. If a matching project exists on the platform, go to the projects panel (or tab) to link it.```

Can you also make "projects panel" link and navigate on click to projects tab in the plugin? It is a bit unclear as we also have projects page in the web app as well.